Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabDomUtils.h

```diff
@@ -21,16 +21,21 @@ namespace AzToolsFramework
         class Instance;
         namespace PrefabDomUtils
         {
-            inline static const char* InstancesName = "Instances";
-            inline static const char* PatchesName = "Patches";
-            inline static const char* SourceName = "Source";
-            inline static const char* LinkIdName = "LinkId";
-            inline static const char* EntityIdName = "Id";
-            inline static const char* EntitiesName = "Entities";
-            inline static const char* ContainerEntityName = "ContainerEntity";
-            inline static const char* ComponentsName = "Components";
-            inline static const char* EntityOrderName = "Child Entity Order";
-            inline static const char* TypeName = "$type";
+            inline static constexpr const char* InstancesName = "Instances";
```

Comment:
nit I missed the first time: prefer `constexpr AZStd::string_view` to cache the string length

Reply:
These const char* are referenced in plenty of places in the code. Since this already is a big PR, I'll do a follow up PR for your suggestion to avoid touching a lot more files and causing potential merge conflicts.